### DISCUSSION TITLE
What did I do wrong?

### DISCUSSION THREAD

**Discussion by Seamoconic** [rustdesk/rustdesk#10127] [General]
Can someone please tell me what I have done wrong?

![image](https://github.com/user-attachments/assets/10d82c51-f610-4a8c-b59f-71842538363c)
from https://github.com/rustdesk/rustdesk/issues/10125#issuecomment-2509516628

-> I have created a discussion (https://github.com/rustdesk/rustdesk/discussions/10039)
-> someone has given me an idea what the error is
-> I found the bug
-> and then I created an issue because I found the bug (https://github.com/rustdesk/rustdesk/issues/10125#issuecomment-2509516628)

Should I not create issues from a Discussion when I have found a bug? 
I really don't know.

You can really take your time to solve the bug. The temporary solution of using an older Rustdesk version works very well for me and maybe for many others. 
I really just wanted to help here but instead I get a very toxic response.

If I did something wrong feel free to block or ban me, I was just trying to help.

Have a nice start into the christmas season.
Mathias

**Comment by Poordeveloper** [comment]
The criminals never think they are wrong.

BTW RDP works fine for me with 1.3.2.

**Comment by Seamoconic** [comment]
I have the feeling that I am still being misunderstood here.

@poordeveloper That's very interesting, maybe it's because I logged on to a domain user on the target PC and you possibly logged on to a local user? It would probably help a lot to fix the bug. But this should rather be discussed here https://github.com/rustdesk/rustdesk/discussions/10039. 

Apparently you know what I did wrong and could have just told me here so that I don't do it again next time instead of accusing me of being a criminal. 

I really don't want to offend anyone here, I just want to help.

But I think I've now found my own mistake and we've really talked past each other here. 

In the company where I work, problems are discussed and if a bug is found, it is always recorded separately as an issue, because only the issues go into the backlog. 

I wanted to save you work so that you don't have to create the issue, but as far as I've seen here now,  issues and discussions are included in the backlog.

As a result, I have created a double post here, which I did not intend to do. 

I'm really sorry, I really didn't mean to do that. 
Sorry for the misunderstanding.
